scripts/codecov.sh

```diff
@@ -0,0 +1,23 @@
+#/bin/bash
+FILENAME=codecov
+curl -s https://codecov.io/bash > $FILENAME
+CODECOV_VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2);
+VALIDATION_FAILURE=0
+for i in 1 256 512
```

Comment:
Doing it three times is probably overkill, a single sha256 should be fine.

Reply:
Removed 1 and 512 checks 
Good point
Done within https://github.com/pagopa/io-app/pull/2988/commits/c21b0d14984d96d3f57529c17d2be573a27c9533